provider/inspec.rb

```diff
@@ -157,5 +159,12 @@ def nested_object_requires(nested_object_type)
         [nested_object_type.__resource.name, nested_object_type.name.underscore].join('_')
       ).downcase
     end
+
+    def plural(word)
```

Comment:
Check out `.pluralize` from the ActiveSupport gem.

ActiveSupport is this crazy set of utils that's a part of Rails.

My only worry is that if you pull in ActiveSupport, you pull in a million things that change every object.

Reply:
There *should* be a way to load only language utils or something. Ruby dependencies are a mystery to me, but I'll try to grab only that.

Reply:
I'm fiddling around with activesupport for a discovery doc related thing right now.

Here's what I've got:

1. `gem install activesupport`
2. `require activesupport/inflector` to only get the grammar related things
3. `"word".pluralize # words`

It's amazing and magical.